cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -34,7 +34,6 @@ import (
 
 	beehiveContext "github.com/kubeedge/beehive/pkg/core/context"
 	"github.com/kubeedge/beehive/pkg/core/model"
-
```

Comment:
you can just remove this line instead.

Reply:
ok, I submitted
 。https://github.com/kubeedge/kubeedge/pull/4304